src/components/QueryEditor/useError.tsx

```diff
@@ -0,0 +1,20 @@
+import { useEffect } from 'react';
+// @ts-ignore
+import appEvents from 'grafana/app/core/app_events';
```

Comment:
This gives me a cringe 😅

Reply:
Yeah not great, added a coment mentioning it's because it's not exported in the sdk yet